src/LuaInterface.cpp

```diff
@@ -43,9 +43,7 @@ LuaInterface::LuaInterface(Host* pH) : mpHost(pH), L(), depth()
     lua_atpanic(interpreter->pGlobalLua, &onPanic);
 }
 
-LuaInterface::~LuaInterface()
-{
-}
+LuaInterface::~LuaInterface() = default;
```

Comment:
Is there any merit in moving ***all*** the `default` cases like this to the header file?

Reply:
Seems like a good idea